apel/db/unloader.py

```diff
@@ -271,6 +279,33 @@ def _write_apel(self, records):
         buf.close()
         del buf
 
+    def _write_yaml(self, records):
+        '''
+        Write one message in the appropriate YAML format to the outgoing
+        message queue.
+        '''
+        buf = StringIO.StringIO()
+        documents = []
+
+        if type(records[0]) == StorageSummaryRecord:
+            documents.append({'type': 'StorageRecord'})
+
+            for record in records:
+                documents.append(record.get_ur(self._withhold_dns))
+        else:
+            raise ApelDbException(
+                "Can only send record's in YAML for StorageRecords"
```

Comment:
```suggestion
                "Can only send records in YAML for StorageRecords"
```
grammar error i think

Reply:
Right. Updated the code now.